automod/triggers.go

```diff
@@ -938,7 +970,7 @@ func (r *MessageRegexTrigger) CheckMessage(triggerCtx *TriggerContext, cs *dstat
 	}
 
 	re := item.Value().(*regexp.Regexp)
-	if re.MatchString(m.Content) {
+	if re.MatchString(m.Content) || dataCast.SanitizeText && re.MatchString(common.SanitizeText(m.Content)) {
```

Comment:
create a separate variable here to store this `common.SanitizeText(m.Content)`

Reply:
Would it be worth to check if SanitizeText is valid and just replace the argument passed to MatchString?